pulsar/io.sls

```diff
@@ -0,0 +1,20 @@
+{% from "pulsar/map.jinja" import pulsar with context %}
+{% set mirror_data = 'https://www.apache.org/dyn/closer.cgi/pulsar?as_json=1'|http_query %}
+{% set mirror_urls = mirror_data.body|load_json %}
+{% set mirror_url = mirror_urls.preferred %}
+{% set mirror_backup = mirror_urls.backup[0] %}
+
+{% for connector in pulsar.io_connectors %}
+install_pulsar_io_connector_{{ connector }}:
```

Comment:
Wouldn't this throw an error if/when `connector` is an empty list?

Reply:
No, because if it's an empty list then it won't iterate at all.